haret/src/vr/states/common/normal.rs

```diff
@@ -67,12 +67,11 @@ pub fn handle_start_view<T: State>(state: T,
 }
 
 pub fn handle_get_state<T: State>(state: T,
-                                  msg: GetState,
+                                  GetState { epoch, view, op }: GetState,
```

Comment:
@mrallen1 we used to have Erlang-style threads, with supervisor notifications when threads died, but we unfortunately ripped that out a long time ago. It imposed a bit thicker of a runtime than what we were aiming for.